core/modules/server/routes/get-tiddlers-json.js

```diff
@@ -31,17 +33,24 @@ exports.handler = function(request,response,state) {
 	var excludeFields = (state.queryParameters.exclude || "text").split(","),
 		titles = state.wiki.filterTiddlers(filter);
 	response.writeHead(200, {"Content-Type": "application/json"});
-	var tiddlers = [];
+	var response_body = [];
 	$tw.utils.each(titles,function(title) {
 		var tiddler = state.wiki.getTiddler(title);
 		if(tiddler) {
 			var tiddlerFields = tiddler.getFieldStrings({exclude: excludeFields});
 			tiddlerFields.revision = state.wiki.getChangeCount(title);
 			tiddlerFields.type = tiddlerFields.type || "text/vnd.tiddlywiki";
-			tiddlers.push(tiddlerFields);
+			response_body.push(tiddlerFields);
+		} else {
+			if(include_non_tiddlers) {
+				let entry = {};
```

Comment:
We stick to ECMAScript 2015 to keep compatibility with older JS environments.

Reply:
Do you mean that `let` should be `var`? I just started to learn javascript so I'm not yet solid in my distinction between the different standards. 
I also tried to seach for ECMAScript 2015 and it seems to support `let` (https://www.ecma-international.org/ecma-262/6.0/#sec-let-and-const-declarations).

Reply:
hmm .. ES2015 is ES6 which would allow `let`and arrow functions.

Reply:
Apologies, I meant to say ES5.1